src/app/core/models/index.ts

```diff
@@ -14,6 +14,7 @@ export const models: any[] = [
   Miscellaneous,
   Tour,
   OrganismPlaceholders,
+  GenePlaceholders,
```

Comment:
`RegionPlaceholders` should be part of this type as well.

Reply:
OK, yes I intended a release for the gcv-docker-compose repo as well- but slightly confused by your comment "That will require updating the GCV submodule"; isn't updating the image tag referenced by compose.prod.yml the only actual change needed?